stock_picking_report_valued/models/stock_move_line.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2014 Pedro M. Baeza - Tecnativa <pedro.baeza@tecnativa.com>
+# Copyright 2015 Antonio Espinosa - Tecnativa <antonio.espinosa@tecnativa.com>
+# Copyright 2016 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2018 Luis M. Ontalba - Tecnativa <luis.martinez@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPackOperation(models.Model):
+    _inherit = "stock.move.line"
+
+    currency_id = fields.Many2one(
```

Comment:
I think we don't need all this overload in the model. As they're related fields and they are not shown in any view, you can use dot notation for accessing its values in the report.

Reply:
I've tried to remove it but test is telling me it is necessary. I think the monetary field needs this one as an auxiliary field

Reply:
Well, I'm talking about all the fields, but OK, let's continue this way.